[18.0] [MIG] product_supplierinfo_archive: Migration to 18.0

Kimkhoi3010 opened the pull request.

clementmbr committed:
[12.0] Add product_packaging_dimension

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Added translation using Weblate (Catalan)

Kev-Roche committed:
[IMP] product_packaging_dimension: black, isort, prettier

Kev-Roche committed:
[MIG] product_packaging_dimension: Migration to 14.0

legalsylvain committed:
[ADD][12.0] new module product_net_weight.
- Add new net_weight field on product models;
- Rename weight field with 'Gross Weight' string;

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
product_packaging_dimension 14.0.1.0.1

GuillemCForgeFlow committed:
[13.0][ADD]product_supplierinfo_archive

manuelregidor committed:
[IMP]product_net_weight: black, isort, prettier

manuelregidor committed:
[MIG]product_net_weight: Migration to 14.0

OCA-git-bot committed:
product_packaging_dimension 14.0.1.0.2

oca-travis committed:
[UPD] Update product_supplierinfo_archive.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

GuillemCForgeFlow committed:
[13.0][FIX]product_supplierinfo_archive: Added field on view and changed filters

flachica committed:
[MIG][15.0] Standard migration product_net_weight

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
product_net_weight 15.0.1.0.1

JasminSForgeFlow committed:
[MIG] product_supplierinfo_archive: Migration to 14.0

JasminSForgeFlow committed:
[MIG] product_supplierinfo_archive: Migration to 15.0

OCA-git-bot committed:
[UPD] README.rst

flachica committed:
[MIG] product_packaging_dimension: 15.0

marielejeune committed:
[MIG] product_packaging_dimension: Migration to 16.0

Commit:
[UPD] Update product_net_weight.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-15.0/product-attribute-15.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-15-0/product-attribute-15-0-product_net_weight/

OCA-git-bot committed:
product_packaging_dimension 16.0.1.0.1

marylla committed:
Added translation using Weblate (German)

victoralmau committed:
[FIX] product_net_weight: Fix digits parameter.

OCA-git-bot committed:
product_net_weight 15.0.1.0.2

carlosdauden committed:
Added translation using Weblate (Spanish)

carlosdauden committed:
Translated using Weblate (Spanish)

Currently translated at 88.8% (8 of 9 strings)

Translation: product-attribute-15.0/product-attribute-15.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-15-0/product-attribute-15-0-product_net_weight/es/

RabbitJon-S73 committed:
[MIG] product_net_weight: Migration to 16.0

sergio-teruel committed:
[IMP] product_net_weight: Adapt module like Odoo manage weight field with multiple variants

PicchiSeba committed:
Added translation using Weblate (Italian)

dsolanki-initos committed:
[IMP] product_supplierinfo_archive: pre-commit stuff

dsolanki-initos committed:
[MIG] product_supplierinfo_archive: Migration to 16.0

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_net_weight/

OCA-git-bot committed:
product_packaging_dimension 16.0.1.0.2

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_net_weight/es/

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update product_supplierinfo_archive.pot

OCA-git-bot committed:
oca-github-bot post-merge updates

JordiBForgeFlow committed:
[FIX] product_packaging_dimension: post-migrate script should only apply query if column max_weight exists.
This column is no longer defined in 16.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_dimension
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_dimension/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_supplierinfo_archive
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_supplierinfo_archive/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_net_weight/it/

CarlosVForgeFlow committed:
[IMP] product_supplierinfo_archive: pre-commit auto fixes

CarlosVForgeFlow committed:
[MIG] product_supplierinfo_archive: Migration to 17.0

Commit:
[UPD] Update product_supplierinfo_archive.pot

OCA-git-bot committed:
[BOT] post-merge updates

nguyenminhchien committed:
[IMP] product_packaging_dimension: pre-commit auto fixes

remihb committed:
[IMP] product_net_weight: pre-commit auto fixes

remihb committed:
[MIG] product_net_weight: Migration to 17.0

nguyenminhchien committed:
[MIG] product_packaging_dimension: Migration to 17.0

Commit:
[UPD] Update product_packaging_dimension.pot

OCA-git-bot committed:
[BOT] post-merge updates

rf-ife committed:
[MIG] product_net_weight: adjust form layout

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[BOT] post-merge updates

LucasTran380381 committed:
[IMP] product_packaging_dimension: pre-commit auto fixes

LucasTran380381 committed:
[MIG] product_packaging_dimension: Migration to 18.0

alexis-via committed:
[IMP] product_net_weight: pre-commit auto fixes for v18 migration

alexis-via committed:
[MIG] product_net_weight to v18

pfranck committed:
[IMP] product_supplierinfo_archive: pre-commit auto fixes

Kimkhoi3010 committed:
[MIG] product_supplierinfo_archive: Migration to 18.0
Co-authored-by: Do Anh Duy <duyda@trobz.com>

nilshamerlinck commented:
please `git commit --amend --author="Franck Plazanet <franck@plazanet.ovh>"` on commit `[IMP] product_supplierinfo_archive: pre-commit auto fixes]`, we try to be polite when taking over existing work

xaviedoanhduy commented on product_supplierinfo_archive/views/product_views.xml:
```diff
@@ -1,31 +1,5 @@
-            </field>
-        </field>
-    </record>
-
```
please restore it, `active` is a special field on form view. deleting it makes form view lose `Archive` action server

![image](https://github.com/user-attachments/assets/75d548b5-c9e9-41d9-b779-463cdcd86da4)

xaviedoanhduy commented:
btw, in the pre-existing PR, add a comment like this: hi, i created (link to new PR) to take over this PR.
![image](https://github.com/user-attachments/assets/83a5e1c0-0e80-4a5b-89c4-7c63f20b2c86)

xaviedoanhduy commented on product_supplierinfo_archive/views/product_views.xml:
```diff
@@ -21,7 +21,7 @@
         <field name="inherit_id" ref="product.product_supplierinfo_form_view" />
         <field name="arch" type="xml">
             <field name="product_variant_count" position="after">
-                <field name="active" invisible="1" />
```
brother, i told you to restore it then you just need to put it back as it was, no need to create more if you haven't tested it that feature. we just need to restore the active field on the form mode with the `invisible="1"` attribute to restore the `Archive` feature on the form view. `active` is a field used to hide records on the view (soft delete) instead of deleting that record (hard delete) and no one wants to show it on the view :)

read more in: https://www.cybrosys.com/blog/what-are-automatic-and-reserved-fields-in-odoo-17

xaviedoanhduy commented on product_supplierinfo_archive/views/product_views.xml:
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_supplierinfo_tree_view" model="ir.ui.view">
+        <field
+            name="name"
+        >product.supplierinfo.tree.view - product_supplierinfo_archive</field>
+        <field name="model">product.supplierinfo</field>
+        <field name="inherit_id" ref="product.product_supplierinfo_tree_view" />
+        <field name="arch" type="xml">
+            <field name="price" position="after">
+                <field name="active" />
+            </field>
+        </field>
+    </record>
```
```suggestion
```

no need invisible on list view

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (51 of 51 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_medical/it/

OCA-git-bot committed:
Merge PR #1787 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update product_logistics_uom.pot

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (31 of 31 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/it/

OCA-git-bot committed:
Merge PR #1752 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update product_packaging_dimension.pot

OCA-git-bot committed:
[BOT] post-merge updates
